### DISCUSSION TITLE
Parallel Computation fails

### DISCUSSION THREAD

**Discussion by ole-alb** [su2code/SU2#2248] [Q&A]
Hello, 
 
I am trying to run SU2 in parallel on a cluster using SLURM. But when I submit my shell script there are two possible errors. 
I used the turb ONERAM6 example from the tutorials.

1. When I run "srun parallel_computation.py -n 128 -f turb_ONERAM6.cfg" I get the following error: FileNotFoundError: [Errno 2] No such file or directory: 'config_CFD.cfg_tmp'
I addition to that the file config_CFD.cfg which SU2 creates, appears but is completely empty. 

2. When I run "parallel_computation.py -n 128 -f turb_ONERAM6.cfg" SU2 just runs 128 times and I get multiple outputs. 

I can't include my whole shell script which I use to execute SU2, but here are the important parts: 

#!/bin/bash
#SBATCH --nodes=2
#SBATCH --ntasks-per-node=64
#SBATCH --ntasks-per-core=1
#SBATCH --time=01:00:00   
#SBATCH --output=[..]/Output-%j.out   
#SBATCH --error=[..]/Error-%j.err

module load su2
module load py-numpy/1.23.4
module load openmpi/4.1.5

export PATH=[...]/bin:$PATH
export SU2_RUN=[...]/bin
export PYTHONPATH=$SU2_RUN:$PYTHONPATH

srun parallel_computation.py -n 128 -f turb_ONERAM6.cfg

I already tried to find a solution to my problem everywhere, but I just don't understand it completely.

Thanks in advance
Ole

**Comment by pcarruscag** [comment]
Have you tried just using mpirun or mpiexec SU2_CFD in the script?
The same output N times is usually because the mpi version you use to compile and run are not the same.
